packages/dynamite/pubspec.yaml

```diff
@@ -6,6 +6,7 @@ environment:
 
 dependencies:
   build: ^2.3.1
+  built_value: ^8.4.4
```

Comment:
This isn't used, is it?

Reply:
I thought I need it for the build.yaml file but turns out I don't